### PR TITLE
Add Jaccard index

### PR DESCRIPTION
**Pull request by ShriprajwalK**
Adds Jaccard index as mentioned in #8

### CODE REVIEW

**Comment by ShriprajwalK** [comment]
Hi @hbollon, I've added code for finding the Jaccard index. Do let me know if I can improve it somehow or if there are any changes to be made.

**Review by hbollon — CHANGES_REQUESTED** [review]
Thanks for your PR! I will review it ASAP
In the meantime, I checked quickly and I noticed that you doesn't added your Jaccard function to `StringSimilarity` one (located in `string-analysis.go`), can you add it please?

**Comment by ShriprajwalK** [comment]
I've added the Jaccard function to `string-analysis.go` and `string-analysis_test.go`. Are the changes made there alright?

File: string-analysis_test.go [hunk 1]
```diff
@@ -139,6 +139,20 @@ func TestStringsSimilarity(t *testing.T) {
 		{"Cosine : Sentence 2", args{"I love horror movies", "Lights out is a horror movie", Cosine}, 0.20412414, false},
 		{"Cosine : Sentence 3", args{"love horror movies", "Lights out horror movie", Cosine}, 0.28867513, false},
 
+		// Jaccard method
+		{"Jaccard : First arg empty", args{"", "abcde", Jaccard}, 0.0, false},
+		{"Jaccard : Second arg empty", args{"abcde", "", Jaccard}, 0.0, false},
+		{"Jaccard : Same args", args{"abcde", "abcde", Jaccard}, 1.0, false},
+		{"Jaccard : No characters match", args{"abcd", "effgghh", Cosine}, 0.0, false},
```
Comment (hbollon) [inline string-analysis_test.go:146]: Idk if this case is useful for Jaccard but anyways it use Cosine algorithm instead of Jaccard one

**Review by hbollon — CHANGES_REQUESTED** [review]

**Comment by hbollon** [comment]
Other than that everything looks good to me :smile: 
However, I must make some modifications to the Ci workflow of this repo and take the time to test the algo before merging.
I'm trying to do this quickly! :wink:

**Comment by ShriprajwalK** [comment]
Made the changes. Thanks a lot!

**Review by hbollon — APPROVED** [review]

**Comment by hbollon** [comment]
It's merged! Thanks for your contribution mate! :smile:

**Pull request merged**